tests/evm/test_calldatasize.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from zkevm_specs.evm import (
+    ExecutionState,
+    StepState,
+    Opcode,
+    verify_steps,
+    Tables,
+    RWTableTag,
+    RW,
+    CallContextFieldTag,
+    Bytecode,
+)
+from zkevm_specs.util import rand_fp, RLC, U64
+from zkevm_specs.util.param import N_BYTES_U64
+
+
+TESTING_DATA = (
+    0x00,
+    0x10,
+    0x302010,
+    0xF0FFFFFFFFFFFF0F,
```

Comment:
This seems to be an impossible value regard to the gas limit in current system (allowing gas only up to `u64`).

Reply:
It should fit inside a u64 no? It's 8 bytes or less

Reply:
What I was thinking is for internal call, when calldata length is so large, it means the memory expansion gas cost already exceeds gas limit `u64`. But I forgot to consider about root call, so let's still have 8 bytes here (though it seem not possible regard to current tx size limit).

Reply:
Oh I put that value without thinking about it too much.  I think it's better to put a sensible value to avoid problems in the future in case we start checking more stuff.  I'll just delete that test vector.